.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -75,87 +96,83 @@ This is a list of questions that need to be answered by the contributor in order
 
 
 ### Target Contributors
-*Describe the contributor profile (background, position, organization) you would like to get contributions from.*
+*Describe the contributor profile (background, position, etc) and if applicable organizational profile (financial institutions, tech vendors, consulting firms, etc) you would like to get contributions from*
 
-## Project Communication Channel(s)
+## Preferred project infrastructure
+If the project is approved, FINOS will provision the infrastructure necessary for [development, release and project collaboration](https://community.finos.org/docs/collaboration-infrastructure). Below a list of questions which will help us accelerate the onboarding process and tailor it to the maintainers needs:
 
-- [ ] Contributor to ask maintainers which communications channels they'd like to use:
-- Asynchronous
-  - [ ] GitHub Issues _(public)_
-  - [ ] GitHub Discussions _(public)_
+- Which code hosting platform does the project prefer? *[Github](https://github.com/finos/)|[GitLab](https://gitlab.com/finos)* 
+- Please check below which asynchronous collaboration channels will the project like to use?
+  - [ ] GitHub Issues
+  - [ ] GitHub Public Discussions
   - [ ] GitHub Team Discussions _(consisting of the above described contributors)_
```

Comment:
GitHub Team Discussions has been deprecated (https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/) we should remove this section.
```suggestion

```